spec/services/step_nav_publisher_spec.rb

```diff
@@ -24,57 +24,23 @@
     context "published step by step having a step edited" do
       let(:step_nav) { create(:published_step_by_step_page) }
 
-      it "defines :auth_bypass_ids in the publishing-api payload" do
+      it "defines :access_limited in the publishing-api payload" do
```

Comment:
This doesn't match with what's being tested (`auth_bypass_ids` on line 30)

Reply:
Good spot, don't know why this changed but have changed back to `auth_bypass_ids`.